src/snowflake/snowpark/context.py

```diff
@@ -8,6 +8,8 @@
 
 _use_scoped_temp_objects = True
 
+#
+
```

Comment:
This will be the place to introduce the hook from Snowflake CLI, as well as have a global flag that allows us to execute code in a "local sandbox" environment, i.e. not interact with Snowflake.